server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionFinalizer.java

```diff
@@ -107,7 +107,8 @@ protected CompactionFinalizer(ServerContext context, ScheduledThreadPoolExecutor
 
   private SharedBatchWriter getWriter(ExternalCompactionId ecid) {
     return writers.computeIfAbsent(ecid.getFirstUUIDChar(),
-        (i) -> new SharedBatchWriter(Ample.DataLevel.USER.metaTable(), context, queueSize));
+        (prefix) -> new SharedBatchWriter(Ample.DataLevel.USER.metaTable(), prefix, context,
+            queueSize / 16));
```

Comment:
When adding the debugging info noticed this queue size is the same for all 16 writers, so divided it by 16.